apps/frontend/src/pages/auth/styles.module.css

```
@@ -30,7 +30,7 @@
 	align-items: center;
 	justify-content: center;
 	width: 454px;
-	min-height: 685px;
+	height: 100%;
```

```suggestion
	height: 100vh;
```
I think if you write min-height in 100vh you should also use this measurements for height attribute